aws/amazon-chime-sdk-component-library-react
# Upgrade storybook version

**Issue #:** 

**Description of changes:**
Try to fix the dependencies security issues.
1. Upgrade `storybook` from 6.29 to 6.49, using `npx sb upgrade`
    ```
    "@storybook/addon-a11y": "^6.4.9",
    "@storybook/addon-docs": "^6.4.9",
    "@storybook/addon-storysource": "^6.4.9",
    "@storybook/addon-viewport": "^6.4.9",
    "@storybook/addons": "^6.4.9",
    "@storybook/react": "^6.4.9",
    "@storybook/storybook-deployer": "^2.8.10",
    "@storybook/theming": "^6.4.9",
    "storybook": "^6.4.9",
    "themeprovider-storybook": "^1.8.0",
    ```
2. add `storybook-static` folder to `.gitignore`. This folder is used to store the built storybook.

**Testing**
1. Have you successfully run `npm run build:release` locally?
Yes

2. How did you test these changes?
* run `npm run build-storybook` and confirm the storybook can be built and it works well.
* run `npm run start` and confirm the storybook works well.

3. If you made changes to the component library, have you provided corresponding documentation changes?
N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
